lib/include/nitro/fx/fx_mtx44.h

```diff
@@ -0,0 +1,82 @@
+#ifndef NITRO_FX_FX_MTX44_H_
+#define NITRO_FX_FX_MTX44_H_
+
+#include <nitro/mi/memory.h>
+#include <nitro/fx/fx.h>
+#include <nitro/fx/fx_const.h>
+
+static inline void MTX_Identity44(MtxFx44 * pDst);
+static inline void MTX_Copy44To33(const MtxFx44 * pSrc, MtxFx33 * pDst);
+static inline void MTX_Copy44To43(const MtxFx44 * pSrc, MtxFx43 * pDst);
+static inline void MTX_Transpose44(const MtxFx44 * pSrc, MtxFx44 * pDst);
+void MTX_TransApply44(const MtxFx44 * pSrc, MtxFx44 * pDst, fx32 x, fx32 y, fx32 z);
+static inline void MTX_Scale44(MtxFx44 * pDst, fx32 x, fx32 y, fx32 z);
+void MTX_ScaleApply44(const MtxFx44 * pSrc, MtxFx44 * pDst, fx32 x, fx32 y, fx32 z);
+static inline void MTX_RotX44(MtxFx44 * pDst, fx32 sinVal, fx32 cosVal);
+static inline void MTX_RotY44(MtxFx44 * pDst, fx32 sinVal, fx32 cosVal);
+static inline void MTX_RotZ44(MtxFx44 * pDst, fx32 sinVal, fx32 cosVal);
+void MTX_RotAxis44(MtxFx44 * pDst, const VecFx32 * vec, fx32 sinVal, fx32 cosVal);
+void MTX_Concat44(const MtxFx44 * a, const MtxFx44 * b, MtxFx44 * ab);
+void MTX_FrustumW(fx32 t, fx32 b, fx32 l, fx32 r, fx32 n, fx32 f, fx32 scaleW, MtxFx44 * mtx);
```

Comment:
the fuck is a t b l r n f

Reply:
please use top bottom left right near far
it's not good enough to just use what nintendo did, this is a decomp, we don't copy and paste from GF's code and we sure as hell shouldn't from nintendo's code
both are not up to the quality we need